File: src/com/example/Recipe_App/SecondActivity.scala
package com.example.Recipe_App

import android.view.View
import android.view.View.OnClickListener
import android.widget.Button

/**
 * Created by jacobpreston on 9/21/14.
 */
class SecondActivity extends BaseActivity {
  onCreate {
    setContentView(R.layout.activity_second_activity)
    val alex_button = findViewById(R.id.login_button).asInstanceOf[Button]
    alex_button.setOnClickListener(new OnClickListener {
      override def onClick(v: View) {
        startActivity[ThirdActvitiy]
      }
    })
  }
}

File: src/com/example/Recipe_App/SecondActivity.java
package com.example.Recipe_App;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

/**
 * Created by jacobpreston on 10/2/14.
 */
public class SecondActivity extends BaseJavaActivity {
    private Button login_button;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_second_activity);
        login_button = (Button)findViewById(R.id.login_button);
        login_button.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(SecondActivity.this, ThirdActvitiy.class);
                startActivity(intent);
            }
        });

    }
}
